clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -755,6 +755,8 @@ private void handleColumnCommand() {
       case CommandActions.DETAILS:
         if (line.hasOption(GravitinoOptions.AUDIT)) {
           newColumnAudit(url, ignore, metalake, catalog, schema, table, column).handle();
+        } else {
+          System.out.println(ErrorMessages.UNSUPPORTED_ACTION);
```

Comment:
need a Main.exit(-1) here

Reply:
Hi @justinmclean , I’ve finished updating the code. Please take a look at the PR again when you have time.